modal-js/src/index.ts

```diff
@@ -34,3 +34,8 @@ export type { ExecOptions, StdioBehavior, StreamMode } from "./sandbox";
 export { ContainerProcess, Sandbox } from "./sandbox";
 export type { ModalReadStream, ModalWriteStream } from "./streams";
 export { Secret, type SecretFromNameOptions } from "./secret";
+export {
+  SandboxFileHandle,
```

Comment:
I think you called this `SandboxFile` in the Go SDK. I think either SandboxFileHandle or SandboxFile is appropriate, but we may want to keep them consistently named unless there's a strong convention one way or another.